### ISSUE TITLE
running on Ubuntu server

### ISSUE THREAD

**Issue by CsepregiArtur** [hassio-addons/addon-wireguard#36]
# Problem/Motivation

I want to set up the addon
## Expected behavior

to work :)
## Actual behavior

s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing... 

-----------------------------------------------------------
 Add-on: WireGuard
 Fast, modern, secure VPN tunnel
-----------------------------------------------------------
 Add-on version: 0.3.0
 You are running the latest version of this add-on.
 System: Ubuntu 18.04.4 LTS  (amd64 / qemux86-64)
 Home Assistant Core: 0.105.5
 Home Assistant Supervisor: 201
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] config.sh: executing... 
[cont-init.d] config.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[10:49:43] INFO: Starting WireGuard...
[#] ip link add wg0 type wireguard
RTNETLINK answers: Not supported
[!] Missing WireGuard kernel module. Falling back to slow userspace implementation.
[#] wireguard-go wg0
WARNING WARNING WARNING WARNING WARNING WARNING WARNING
W                                                     G
W   You are running this software on a Linux kernel,  G
W   which is probably unnecessary and misguided. This G
W   is because the Linux kernel has built-in first    G
W   class support for WireGuard, and this support is  G
W   much more refined than this slower userspace      G
W   implementation. For more information on           G
W   installing the kernel module, please visit:       G
W           https://www.wireguard.com/install         G
W                                                     G
WARNING WARNING WARNING WARNING WARNING WARNING WARNING
INFO: (wg0) 2020/02/24 10:49:43 Starting wireguard-go version 0.0.20200121
[#] wg setconf wg0 /dev/fd/63
[#] ip -4 address add 172.27.66.1/24 dev wg0
[#] ip link set mtu 1420 up dev wg0
[#] resolvconf -a wg0 -m 0 -x
[#] iptables -A FORWARD -i wg0 -j ACCEPT; iptables -A FORWARD -o wg0 -j ACCEPT; iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE
## Steps to reproduce

instal change addres and error

**Comment by tjorim** [comment]
Hi, where do you see errors? There is only a warning message that explains everything you need to know. You can just use it like this but the wireguard connection might be slower.

**Comment by frenck** [comment]
There is no error in the logs you posted.

**Comment by CsepregiArtur** [comment]
Thank you for your support, I think this is my error.
I followed frenck video of how to connect.
everything seems to be fine, but homassistant.local is not working, in my case didn't work til now, I used the IP address, now I'm trying with the IP, not working, not with my local address HTTPS://192.168.1.xx and not with the https://172.27.66.1:8123,  I the log from my iphone found [NET] peer(r69U...uEFs) - Handshake did not complete after 5 seconds, retrying (try 8) and then is starting from try 2.
